release/models/network-instance/openconfig-network-instance.yang

```diff
@@ -776,6 +789,11 @@ module openconfig-network-instance {
           }
         }
 
+        uses oc-flexalgo:fad-definition-top {
+	  description
+ 	    "List of Flexible Algorithms and their definitions";
+	}
+
```

Comment:
Instead of adding flexalgo via a uses statement, it is preferred to use 'augment' in the flexalgo model.  This helps prevent unbounded growth of the  network-instance-model file and is convenient for implementors who make not support or choose to make the flexalgo model optional.